WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,258 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, that implements trustless CoinJoin over the Tor anonymity network.">
+    <meta http-equiv="onion-location" content="http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion" />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://wasabiwallet.io">
+    <meta property="og:title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+    <meta property="og:description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, it implements trustless CoinJoin over the Tor anonymity network.">
+    <meta property="og:image" content="https://wasabiwallet.io/images/wasabi-social.jpg">
```

Comment:
Do not harcode domains. Can this use a relative path?

Reply:
For meta we need to hardcode them.

Reply:
Why. Nothing is required to be hardcoded. Why is this case special?

Reply:
These social meta tags just for sharing the website in social media. When you copy and paste the url the social media site for ex Facebook pull data from these tags. If I write just /images/wasabi-social.jpg the image won't show. We don't have php or js what fill the tags with the url when the site loaded.

Reply:
Thanks for the explanation. These meta cannot be here because they are not constant (they depend on domain) and i am sure we don't want to have them repeated in many places.

Reply:
Ok I understand, it is already wrong and you copied and pasted. That doesn't mean this has to continue being the case forever.

Reply:
The server can inject these meta tags. In that way we can remove them from all the pages. By doing that we can inject the right content into the right places. For example, while in production is has to be `<meta property="og:url" content="https://wasabiwallet.io">` in testing it should be `<meta property="og:url" content="https://wasabiwallet.co">`. The same for the onion address, while in production is has to be `<meta http-equiv="onion-location" content="http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion" />` in testing it should be `<meta http-equiv="onion-location" content="http://testwnp3fugjln6vh5vpj7mvq3lkqqwjj3c2aafyu7laxz42kgwh2rad.onion/" />`.

This is important not only for correctness but also for privacy because users don't know (i didn't know neither until now) that by visiting the testing you are hitting the production server for icons and things like that.